addons/website_event/views/website_event_design_registration_badge.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <template id="report_design_registration_badge">
```

Comment:
Should be in `report/` directory, in a file named `report/event_event_templates.xml`.